src/atopile/parser/AtoParser.g4

```diff
@@ -28,7 +28,8 @@ simple_stmt
 
 compound_stmt: blockdef;
 
-blockdef: blocktype name ('from' name_or_attr)? ':' block;
+blockdef: blocktype name blockdef_super? ':' block;
+blockdef_super: 'from' name_or_attr;
```

Comment:
This is split out so we can provide a terser traceback (eg. highlight that it was the super that pulled the code in)